psi4/src/read_options.cc

```diff
@@ -1576,6 +1576,29 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_str("UHF_NOONS", "3");
         /*- Save the UHF NOs -*/
         options.add_bool("SAVE_UHF_NOS", false);
+
+        /*- SUB-SUBSECTION TDDFT -*/
+
+        /*- Get the number of desired states per irrep -*/
+        options.add("TDSCF_STATES_PER_IRREP", new ArrayType());
+        /*- Controls inclusion of triplet states, valid options:
+            - none : No triplets computed
+            - also : lowest-energy triplets and singlets included
+            - only : Only triplet states computed
+             -*/
+        options.add_str("TDSCF_TRIPLETS", "none");
+        /*- Run with Tamm-Dancoff approximation, uses RPA when false -*/ 
+        options.add_bool("TDSCF_TDA", false);
+        /*- Convergence threshold for excitation energies -*/
+        // NOTE: This option may do nothing
+        options.add_double("TDSCF_E_TOL", 1E-6);
```

Comment:
Is this and below playing the `E/R_CONVERGENCE` roles? Need to have those names, then, with TD* prefix. In practice, should take TD_E/R_CONVERGENCE if has_changed, else E/R_CONVERGENCE values.

Reply:
I plan on entirely phasing out the use of `e_tol` in the solver with #1814 Only `r_tol` should remain as the main convergence check. And the way convergence thresholds propagate should go the other way around: converging the residual of the response equations to 10^{-N} requires convergence of SCF to 10^{-(N+2)} (as for gradients, in practice).

There are other ways to tweak convergence of the response solver: either by starting with more trial vectors in the subspace (in practice ask for more roots than you actually need) or by allowing the subspace to become "arbitrarily" large during the iterations (`TDSCF_MAX_SS_VECTORS`)

Reply:
I think kill off `TDSCF_E_TOL` and replace `TDSCF_R_TOL` with `TDSCF_R_CONVERGENCE`.